ngraph/test/util/engine/ie_engines.cpp

```diff
@@ -172,6 +214,32 @@ testing::AssertionResult test::IE_Engine::compare_results(const size_t tolerance
     return comparison_result;
 }
 
+std::string test::IE_Engine::get_output_name(const std::shared_ptr<op::v0::Result>& ng_result)
+{
+    if (m_function->get_results().size() == 1)
+    {
+        // ng_result argument is ignored
+        return m_network_outputs.begin()->first;
+    }
+    else
```

Comment:
Do test_case support networks with multiple outputs?
I know there were some problems with it.

Reply:
It works for multiple outputs. The logic of calculating output name is the same as before. I moved it only to helper method.
For single output `m_network_outputs.begin()->first;` is always returned (result argument is not used).
For multioutput case the name is based on previous node name (plus dot).